_posts/2016-11-09-Rust-1.13.md

```diff
@@ -0,0 +1,375 @@
+---
+layout: post
+title: "Announcing Rust 1.13"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.13.0. Rust is
+a systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.13.0][install] from the appropriate page on
+our website, and check out the [detailed release notes for 1.13.0][notes] on
+GitHub. 1448 patches were landed in this release.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-1130-2016-11-10
+
+### What's in 1.13 stable
+
+It's been a really busy season in Rust. We just powered through three Rust
+conferences in short succession, and all the while we've been hacking
+away. There's a whole bunch of amazing stuff coming down the pipeline, but in
+today's release we're highlighting three new language features, including the
+long-awaited `?` operator, as well as a surge in progress on compile-time
```

Comment:
Talking about a "surge in progress" is complicated here, because you want to talk about progress increasing on making something decrease, which sounds weird. The first time I read this, I thought this meant that compile times increased.

Maybe "as well as significant speed improvements in compile times."